.travis.yml

```diff
@@ -5,7 +5,7 @@ sudo: false
 install:
   - nvm install 6
   - nvm use 6
-  - npm install -g purescript@0.10.2 pulp bower
+  - npm install -g purescript@0.10.7 pulp@10.0.4 bower
```

Comment:
That doesn't work for me.

They don't seem to be on npm yet- is this still the correct package? https://www.npmjs.com/package/purescript